src/content/user-guide/deepsparse-engine/logging.mdx

```diff
@@ -0,0 +1,319 @@
+---
+title: "Logging"
+metaTitle: "DeepSparse Logging"
+metaDescription: "System and Data Logging with DeepSparse"
+index: 6000
+---
+
+# DeepSparse Logging
+
+DeepSparse Logging provides you with access to telemetry needed to monitor a deployment. 
+
+There are many types of downstream monitoring tasks that you may want to perform. The difficulty of the tasks varies from relatively easy 
+(simple system performance analysis) to challenging (assessing the accuracy of the system in the wild by manually labeling 
+the input data distribution post-facto). Examples include:
+- System performance: what is the latency/throughput of a query?
+- Data quality: is there an issue getting data to my model?
```

Comment:
```suggestion
- Data quality: is the model being fed the expected data distribution?
```

Reply:
Data quality is about detecting errors in the data pipeline. For instance, detecting if the camera is broken and as a result you are getting all 0's. Vs data distribution is slightly different and we have a separate bullet for that